### PR TITLE
M: https://www.heise.de/

### PR DESCRIPTION
**Pull request by salim-b**
Hide Ad containers.

Site to test: https://www.heise.de/

### CODE REVIEW

**Comment by Khrin** [comment]
Ok, I see. I don't think these links should be blocked then, because are considered as self-promotion, as per EasyList Germany policy. Sorry

**Comment by salim-b** [comment]
> I don't think these links should be blocked then, because are considered as self-promotion, as per EasyList Germany policy.

I don't want to be a dick, but I think *some* of them actually are. The first highlighted box above links to a post on the heise microsite [„Die Zukunft der Arbeit“](https://it-kenner.heise.de/zukunft-der-arbeit/). In the footer of that site it says:

> Diese Microsite „Die Zukunft der Arbeit“ wird betrieben von [Heise Medien GmbH & Co. KG](https://www.heise.de/impressum.html) in Zusammenarbeit mit [MBmedien Group GmbH](https://mbmedien.group/impressum) **mit freundlicher Unterstützung durch [Dell Technologies](http://www.delltechnologies.com/), [Microsoft](https://www.microsoft.com/de-de/rechtliche-hinweise/impressum) und [NVIDIA.](https://nvidianews.nvidia.com/)**.

The current EasyList rules seem to [define self-promo](https://github.com/easylist/easylist#user-content-fn-1-edc788fbd4fd43fdfb33cb29b270b17e) as

> Self-promotion is any type of advertising that promotes goods or services that are owned or operated by the domain owner and **doesn’t get commercially compensated for by third parties** (examples can include new features, new posts, newsletters, subscriptions to printed media products, merchandise etc.)

The second highlighted box above OTOH would actually fall into that self-promo category. And it would probably be very difficult (or even impossible) to only block the first but not the second box via simple content blocker filter rules.

tl;dr: **Would the filter rule of this PR meet the acceptance criteria for [Fanboy's Annoyance List](https://github.com/easylist/easylist/tree/master/fanboy-addon)?**

**Comment by Khrin** [comment]
@salim-b Don't worry, you're not a "dick". Actually you've raised a good point here, and I think this filter can help to block these links. (The links towards to "_heise+_" cannot be blocked)
